src/client/components/ListOfWeeksWithDetail/index.js

```diff
@@ -28,6 +31,8 @@ function ListOfWeeksWithDetail(props) {
 
 ListOfWeeksWithDetail.propTypes = {
   userData: userDataPropType.isRequired,
+  showContent: PropTypes.arrayOf(PropTypes.bool).isRequired,
```

Comment:
Maybe call this showContents or showContentsArray to differentiate it from showContent in AccordianItem.

Reply:
good point, I didn't notice that!